site/specs-temp/numbers.json

```diff
@@ -19965,6 +19965,7 @@
         "type": "object",
         "properties": {
           "TelephoneNumber": {
+            "description": "Simple Telephone Number",
```

Comment:
For all of the descriptions, it looks like there is no period at the end of the descriptions. These should be normal sentences and it would be awesome if we could add periods to all the descriptions below to match customer expectations.